CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/docs/notes-hs.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE helpset PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp HelpSet Version 2.0//EN" "http://java.sun.com/products/javahelp/helpset_2_0.dtd">
+<helpset version="2.0">
+    <title>Constellation</title>
+    <maps>
+        <homeID>au.gov.asd.tac.constellation.views.layers.about</homeID>
```

Comment:
I believe this should say 
`<homeID>au.gov.asd.tac.constellation.views.notes.about</homeID>`

Reply:
Good spot. Will update